src/pages/evm/wallet/WalletPageHeader.vue

```diff
@@ -0,0 +1,152 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+import InlineSvg from 'vue-inline-svg';
+
+import BigFiatText from 'components/evm/BigFiatText.vue';
+import { useChainStore } from 'src/antelope';
+import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
+
+export default defineComponent({
+    name: 'WalletPageHeader',
+    components: {
+        BigFiatText,
+        InlineSvg,
+    },
+    computed: {
+        // TODO return real balance
+        // https://github.com/telosnetwork/telos-wallet/issues/176
+        totalBalance() {
+            return 123456.78;
+        },
+        buyMoreLink() {
+            const chainStore = useChainStore();
+            return (chainStore.currentChain.settings as EVMChainSettings).getBuyMoreOfTokenLink();
+        },
+    },
+    methods: {
+        goToBuy() {
+            window.open(this.buyMoreLink, '_blank')?.focus();
+        },
+    },
+});
+</script>
+
+<template>
+<div>
+    <BigFiatText :amount="totalBalance" class="q-mb-xl"/>
+
+    <div class="c-wallet-page-header__links">
+        <div
+            class="c-wallet-page-header__link"
+            tabindex="0"
+            role="link"
+            :aria-label="$t('evm_wallet.link_to_send_aria')"
+            @keydown.space.enter="$router.push({ name: 'evm-send' })"
```

Comment:
can maybe extract this here and in the other divs to something like 
methods: {
  navigateTo(routeName) {
    this.$router.push({ name: routeName });
  },
},